hs_core/migrations/0053_auto_20200826_1629.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-08-26 16:29
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hs_core', '0052_tasknotification_username'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='tasknotification',
+            name='status',
+            field=models.CharField(choices=[('progress', 'Progress'), ('failed', 'Failed'), ('aborted', 'Aborted'), ('completed', 'Completed'), ('delivered', 'Delivered')], default='pending', max_length=20),
+        ),
```

Comment:
default cannot be 'pending' any more now that you removed 'pending' from the choices list.

Reply:
nice catch